build/nv.d3.js

```diff
@@ -9593,179 +9603,1064 @@ nv.models.multiChart = function() {
 
-            container
-                .on('click', function(d,i) {
-                    dispatch.chartClick({
-                        data: d,
-                        index: i,
-                        pos: d3.event,
-                        id: id
-                    });
-                });
 
-            defsEnter.append('clipPath')
-                .attr('id', 'nv-chart-clip-path-' + id)
-                .append('rect');
+    var charts = [lines1, lines2, lines3, lines4, scatters1, scatters2, scatters3, scatters4, bars1, bars2, stack1, stack2];
 
-            wrap.select('#nv-chart-clip-path-' + id + ' rect')
-                .attr('width', availableWidth)
-                .attr('height', availableHeight);
 
-            g   .attr('clip-path', clipEdge ? 'url(#nv-chart-clip-path-' + id + ')' : '');
+    function chart(selection) {
+        selection.each(function(data) {
+            var container = d3.select(this),
+                that = this;
+            nv.utils.initSVG(container);
 
-            var ticks = wrap.select('.nv-ticks').selectAll('.nv-tick')
-                .data(function(d) { return d });
-            ticks.exit().remove();
+            var brush = d3.svg.brush()
+                .x(x2)
+                .on("brush", onBrush);
 
-            ticks.enter().append('path')
-                .attr('class', function(d,i,j) { return (getOpen(d,i) > getClose(d,i) ? 'nv-tick negative' : 'nv-tick positive') + ' nv-tick-' + j + '-' + i })
-                .attr('d', function(d,i) {
-                    return 'm0,0l0,'
-                        + (y(getOpen(d,i))
-                            - y(getHigh(d,i)))
-                        + 'l'
-                        + (-w/2)
-                        + ',0l'
-                        + (w/2)
-                        + ',0l0,'
-                        + (y(getLow(d,i)) - y(getOpen(d,i)))
-                        + 'l0,'
-                        + (y(getClose(d,i))
-                            - y(getLow(d,i)))
-                        + 'l'
-                        + (w/2)
-                        + ',0l'
-                        + (-w/2)
-                        + ',0z';
-                })
-                .attr('transform', function(d,i) { return 'translate(' + x(getX(d,i)) + ',' + y(getHigh(d,i)) + ')'; })
-                .attr('fill', function(d,i) { return color[0]; })
-                .attr('stroke', function(d,i) { return color[0]; })
-                .attr('x', 0 )
-                .attr('y', function(d,i) {  return y(Math.max(0, getY(d,i))) })
-                .attr('height', function(d,i) { return Math.abs(y(getY(d,i)) - y(0)) });
+            chart.update = function() { 
+                container.transition().call(chart); 
+                var wrapper = d3.select('svg.nvd3-svg')[0][0].parentNode;
+                d3.select(wrapper).select('svg:nth-child(3)').remove();
+                d3.select(wrapper).select('svg:nth-child(4)').remove();
+            };
```

Comment:
Add TODO comment to explain this workaround, and ideally find a solution that does not require this hack.